src/debugger.ts

```diff
@@ -64,6 +70,10 @@ export namespace Debugger {
       return this._isDisposed;
     }
 
+    get notebook() {
```

Comment:
This seems to return a `NotebookTracker` instead of `Notebook` (can lead to some confusion for users of this API)